src/babylon.engine.ts

```diff
@@ -1401,7 +1408,10 @@
 
             if (this._activeRenderLoops.length > 0) {
                 // Register new frame
-                this._frameHandler = Tools.QueueNewFrame(this._bindedRenderFunction, this._vrDisplayEnabled);
+                var requester = window;

```

Comment:
Would be better to keep `window` reference to a minimum, especially due to https://github.com/BabylonJS/Babylon.js/issues/2529

Reply:
perhaps I could just pass null in to the function instead, so it defaults?

Reply:
would be great, but shouldn't prevent merging.